build.sbt

```diff
@@ -1,11 +1,11 @@
 import akka._
 import akka.ValidatePullRequest._
 import AkkaDependency._
-import Dependencies.{ h2specName, h2specExe }
+import Dependencies.{h2specExe, h2specName}
```

Comment:
surprised this didn't trigger scalariform - I guess it doesn't check the build.sbt?